app/controllers/proposals_controller.rb

```diff
@@ -74,6 +75,17 @@ def update
     end
   end
 
+  def toogle_favorite
+    user = User.find(params[:favourite_user_id])
+    users = @event.favourite_users
+    if users.include? user
```

Comment:
According to Ruby style guide it seems that parenthesis are preferred: https://github.com/bbatsov/ruby-style-guide#method-invocation-parens